.github/workflows/main.yml

```diff
@@ -32,18 +36,33 @@ jobs:
         include:
           - ros-distro: noetic
             ros-version: 1
+            platform: "amd64"
           - ros-distro: galactic
             ros-version: 2
+            platform: "amd64"
           - ros-distro: humble
             ros-version: 2
+            platform: "amd64"
             base-image: "ubuntu:22.04"
-    env:
-      REGISTRY: ghcr.io
+    outputs:
+      key: ${{ steps.artifacts.outputs.key }}
+      tag: ${{ steps.script.outputs.tag }}
     steps:
       - uses: actions/checkout@v3
+      - name: Login to Docker registry
+        uses: docker/login-action@v2
+        with:
+          registry: ${{ env.REGISTRY }}
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+      - name: Install QEMU
+        run: |
+          sudo apt update -qq
+          sudo apt install -qq -y qemu qemu-user-static
```

Comment:
This is on the standard Ubuntu installation without custom dependencies so if the step fails, it is very clear because of which package. In this case, having `-qq` makes sense.

Reply:
This is only needed when generating ARM64